custom/icds_reports/utils/aggregation_helpers/monolith/thr_froms_ccs_record.py

```diff
@@ -2,15 +2,25 @@
 from __future__ import unicode_literals
 
 from dateutil.relativedelta import relativedelta
+
 from custom.icds_reports.const import AGG_CCS_RECORD_THR_TABLE
-from custom.icds_reports.utils.aggregation_helpers import BaseICDSAggregationHelper, month_formatter
+from custom.icds_reports.utils.aggregation_helpers import month_formatter
+from custom.icds_reports.utils.aggregation_helpers.monolith.base import BaseICDSAggregationHelper
 
 
 class THRFormsCcsRecordAggregationHelper(BaseICDSAggregationHelper):
     ucr_data_source_id = 'static-dashboard_thr_forms'
     aggregate_parent_table = AGG_CCS_RECORD_THR_TABLE
     aggregate_child_table_prefix = 'icds_db_ccs_thr_form_'
 
+    def aggregate(self, cursor):
```

Comment:
Is it possible to use mix-ins so that you don't need to repeat identical `aggregate()` methods?

Reply:
yes 100%, that's what I commented in the description.